lib/Quantization/Quantization.cpp

```diff
@@ -179,6 +179,11 @@ class FunctionQuantizer : public FunctionConverter {
       return (SLS->getData().getElementType() == ElemKind::FloatTy) &&
              (SLS->getWeights().getElementType() == ElemKind::FloatTy);
     }
+    case Kinded::Kind::SelectNodeKind: {
+      auto *SN = cast<SelectNode>(&node);
+      return SN->getLHS().getElementType() == ElemKind::FloatTy &&
```

Comment:
@jfix71 Do you really need `FloatTy` here or should any FP type be OK? If so, we have a predicate to check that it is one of FP types.

Reply:
It could be any FP type, however we assume it's FloatTy everywhere here in the FunctionQuantizer. Might be good for a separate PR to make the change globally in the FunctionQuantizer.